arch/arm/core/aarch64/isr_wrapper.S

```diff
@@ -83,8 +82,6 @@ spurious_continue:
 
 	/* --(_kernel->nested) */
```

Comment:
```suggestion
	/* if(!--(_kernel->nested)) { */
```

Nit: perhaps update the comment now that it's not nearly as clear what the `bne` below is doing?